domain/build.gradle

```diff
@@ -0,0 +1,77 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+apply from: '../config/jacoco.gradle'
+
+def globalConfig = rootProject.extensions.getByName("ext")
+android {
+    compileSdkVersion globalConfig.getAt("android_compile_sdk_version")
+    defaultConfig {
+        minSdkVersion globalConfig.getAt("android_min_sdk_version")
+        targetSdkVersion globalConfig.getAt("android_target_sdk_version")
+        versionCode globalConfig.getAt("android_version_code")
+        versionName globalConfig.getAt("android_version_name")
+    }
+
+    flavorDimensions "stage"
```

Comment:
So, `stage` is referring to `staging`?

Reply:
@Tuubz this is `flavorDimensions`. In multiple modules android project, to make a constraint on build flavor between modules, we need to define and use the same flavorDimensions

Reply:
Got it, but in the future we should preferably rename it to `staging`? 😋

Reply:
@Tuubz nooo lol, `stage` here doesn't mean `staging`, actually its name should be `flavor` or something like that
```
flavorDimensions "stage"
    productFlavors {
        production {
            dimension "stage"
        }

        staging {
            dimension "stage"
        }

        development {
            dimension "stage"
        }
    }
```